From: Chow Loong Jin <hyperair@debian.org>
Date: Thu, 16 Feb 2012 02:28:40 +0800
Subject: Amend InternalsVisibleTo directives

Replace PublicKey=$hash with PublicKey=null since we're no longer signing
assemblies.
---
 .../Mono.Upnp.Dcp.MediaServer1/AssemblyInfo.cs     |   10 +---------
 src/Mono.Upnp/Mono.Upnp/AssemblyInfo.cs            |   21 ++------------------
 2 files changed, 3 insertions(+), 28 deletions(-)

diff --git a/src/Mono.Upnp.Dcp/Mono.Upnp.Dcp.MediaServer1/Mono.Upnp.Dcp.MediaServer1/AssemblyInfo.cs b/src/Mono.Upnp.Dcp/Mono.Upnp.Dcp.MediaServer1/Mono.Upnp.Dcp.MediaServer1/AssemblyInfo.cs
index 3f118b5..fc173c0 100644
--- a/src/Mono.Upnp.Dcp/Mono.Upnp.Dcp.MediaServer1/Mono.Upnp.Dcp.MediaServer1/AssemblyInfo.cs
+++ b/src/Mono.Upnp.Dcp/Mono.Upnp.Dcp.MediaServer1/Mono.Upnp.Dcp.MediaServer1/AssemblyInfo.cs
@@ -49,12 +49,4 @@ using System.Runtime.CompilerServices;
 // if desired. See the Mono documentation for more information about signing.
 
 [assembly: AssemblyDelaySign (false)]
-[assembly: InternalsVisibleTo ("Mono.Upnp.Dcp.MediaServer1.Tests, PublicKey=" +
-                               "00240000048000009400000006020000002400005253" +
-                               "41310004000011000000C3030A2933D77BEDEF4CB040" +
-                               "07AF2DFBAEC11AB77693AD580B6B6900B1441006A8FE" +
-                               "CA691E010996573F3DF3E273F4589468125D446128DF" +
-                               "93A1CEEA9407A948990CE3DE2E5531BF4D3063ECA91E" +
-                               "10D4C84B6A77FCDC88F0FDEB9A0FED16D866687DACB8" +
-                               "5659DB48FD017FD94222433D9C453813A591F4E1CF25" +
-                               "CFD2ABFBFE94")]
+[assembly: InternalsVisibleTo ("Mono.Upnp.Dcp.MediaServer1.Tests")]
diff --git a/src/Mono.Upnp/Mono.Upnp/AssemblyInfo.cs b/src/Mono.Upnp/Mono.Upnp/AssemblyInfo.cs
index 264b6c3..4973419 100644
--- a/src/Mono.Upnp/Mono.Upnp/AssemblyInfo.cs
+++ b/src/Mono.Upnp/Mono.Upnp/AssemblyInfo.cs
@@ -25,22 +25,5 @@ using System.Runtime.CompilerServices;
 // if desired. See the Mono documentation for more information about signing.
 
 [assembly: AssemblyDelaySign (false)]
-[assembly: InternalsVisibleTo ("Mono.Upnp.Dcp.MediaServer1, PublicKey=" +
-                               "00240000048000009400000006020000002400" +
-                               "0052534131000400001100000007CA7138D878" +
-                               "FFB4E68E1222C8FD84557E0336F4FB76D83774" +
-                               "13CC48DAB29CE14BEA65AAB4250A04FB059F39" +
-                               "CCE2593DC29ADAF8954E922F4A6354B6DEB71E" +
-                               "909137B33DEA212BB04F1492FBDFE50B0B7614" +
-                               "2974F2E5D86049D7F44706839CD76C57FAC02E" +
-                               "86C977188DD5185F5DDBA8EE46CA8F6F40224F" +
-                               "9D8653092BBAFE9F")]
-
-[assembly: InternalsVisibleTo ("Mono.Upnp.Dcp.MediaServer1.FileSystem, PublicKey=" +
-                               "0024000004800000940000000602000000240000525341310" +
-                               "0040000110000007FD08FD4F03157DEE6573823B967BCF78D" +
-                               "A200777DF681030D1C8EC06BEA3BE2C9CEC6735CF0046953E" +
-                               "1949882B0FB1D6F13ED43A72465BDA6EB3DA794D98D3921AF" +
-                               "D24996E2FFE2FCF87B2B715DA8F20363314259C8F280E71C7" +
-                               "46781846E73D54C383DC32B8C19038F317F9394815C0307BE" +
-                               "92D0BA9CB278C2019B31A28D83")]
+[assembly: InternalsVisibleTo ("Mono.Upnp.Dcp.MediaServer1")]
+[assembly: InternalsVisibleTo ("Mono.Upnp.Dcp.MediaServer1.FileSystem")]
-- 
